service_capacity_modeling/models/org/netflix/kafka.py

```diff
@@ -503,8 +507,12 @@ def default_desires(user_desires, extra_model_arguments: Dict[str, Any]):
         retention = extra_model_arguments.get("retention", "PT8H")
         retention_secs = iso_to_seconds(retention)
 
-        # write throughput * retention = usage
-        state_gib = (write_bytes.mid * retention_secs) / GIB_IN_BYTES
+        # write throughput * retention * replication factor = usage
+        state_gib = (
+            write_bytes.mid
+            * retention_secs
+            * NflxKafkaCapacityModel.HA_DEFAULT_REPLICATION_FACTOR
```

Comment:
can use `extra_model_arguments.get("cluster_type", "high-availability")` to derive the replication factor for default_desires